ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml

```diff
@@ -467,6 +467,18 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: aws-ipi-p2-f28-long-duration-apiserver
+  cron: 29 18 * * 5
```

Comment:
You can use [script](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh) to generate cron entries, which will spread the jobs across the day/month.

Reply:
Thanks, Fixed!